src/main/java/com/monumental/services/AwsS3Service.java

```diff
@@ -0,0 +1,72 @@
+package com.monumental.services;
```

Comment:
I realize that we might not have a need for SDK integration with S3 on the back-end, but I had already made this service class and figured it might be useful for something.

If it turns out it isn't, I can remove it.

Reply:
Now that I think about it more, this might be useful for bulk uploading with images.

Reply:
Bulk uploading is going to happen on the front end, same as single upload, and we would still end up uploading images twice (once to our server, then once from our server to s3) if we do it this way. I think it just doubles the upload time for no good reason when we could just simply upload to s3 from the client (which AWS provides a secure way to do).

Reply:
Alright, I'll remove the `storeObject()` function.

Reply:
Once my other PR gets merged that moved all of the classes in `/util` and added the `StringHelper` class, I'll move `getObjectUrl` into that class.

Reply:
Done.